lib/util/i18n.js

```diff
@@ -0,0 +1,21 @@
+import objectPath from 'object-path'
+
+// Load the localized strings for all languages here.
+// (We cannot really load languages on demand using fetch... that would require tinkering with mastarm build,
+// so here we build an index of translated messages.)
+const translatedMessages = {
+  'en-US': require('../../i18n/en-US.yml'),
```

Comment:
One thing that stands out to me here is that we may need to sub out these strings on a per-implementation basis. E.g., the names of otp modes in Portland OR may be different than in Atlanta (e.g. light rail vs streetcar for TRAM). Not really a requirement for right now, but any ideas for this at the moment?

Reply:
I agree that this is needed, but think it'd be better to figure this out now rather than later. Also, I think it should be possible to load language files on-demand using fetch - the app should just wait until the base language file is loaded before the first main render. I think it'd make a lot of sense to have some kind of config file that can be used to override a default set of language files, however that process would likely be part of the trimet-mod-otp project (at least until we merge those two projects together).